.github/workflows/automerge.yml

```diff
@@ -0,0 +1,37 @@
+name: Automerge
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  # replace "github_username" with your GitHub username
+  # replace "github.com/username/repo.git" with your GitHub repo path
+  # do NOT replace ${{secrets.GITHUB_TOKEN}}, GitHub will take care of it
+  MY_REPO: https://GeospatialResearch:${{secrets.GITHUB_TOKEN}}@github.com/GeospatialResearch/Digital-Twins-191.git
+
+  # replace "long-lived_branch_name" with your branch name
+  LONG_LIVED_BRANCH: master
+
+  # replace it with the path to master repo
+  MASTER_REPO: https://github.com/GeospatialResearch/Digital-Twins-191.git
+
+  # replace "master" with your master branch name
+  DEFAULT_BRANCH: otakaro_digital_twin
```

Comment:
These comments all state what you should do with the template, but we aren't working from the template anymore so it's better that they describe what each of these variables are